src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2164,4 +2164,10 @@ From there, simply click on one of the supported files in the File Explorer and
   <data name="AlwaysOnTop_RoundCorners.Content" xml:space="preserve">
     <value>Enable round corners</value>
   </data>
-</root>
+  <data name="FileExplorerPreview_ToggleSwitch_Monaco_Try_Format.Description" xml:space="preserve">
+    <value>Applies to json and xml. Files are unchanged.</value>
```

Comment:
```suggestion
    <value>Applies to json and xml. Files keep unchanged.</value>
```

Incorrect verb. `are` => `keep`

Reply:
> ```suggestion
>     <value>Applies to json and xml. Files keep unchanged.</value>
> ```
> 
> Incorrect verb. `are` => `keep`

Disagree. "keep" means to hold on to something. "Stay" would be a good alternative.

Reply:
Hmmm. Yep. My mistake.

Reply:
"Remain" is even better IMO